zeppelin-zengine/src/main/java/org/apache/zeppelin/helium/HeliumBundleFactory.java

```diff
@@ -215,29 +215,28 @@ public File getHeliumPackageBundleCache(String pkgName) {
     return new File(heliumBundleDirectory, pkgName + "/" + HELIUM_BUNDLE_CACHE);
   }
 
-  public static List<String> unTgz(File tarFile, File directory) throws IOException {
-    List<String> result = new ArrayList<String>();
-    InputStream is = new FileInputStream(tarFile);
-    GzipCompressorInputStream gcis = new GzipCompressorInputStream(is);
-    TarArchiveInputStream in = new TarArchiveInputStream(gcis);
-    TarArchiveEntry entry = in.getNextTarEntry();
-    while (entry != null) {
-      if (entry.isDirectory()) {
+  private static List<String> unTgz(File tarFile, File directory) throws IOException {
```

Comment:
Well I did change it to private because this method was only used in this class. 
In this last commit I took a look at other methods are reduced visibility when applicable. Kinda of unrelated with the bug itself but does not hurt in my opinion.